pkg/controller/tenant/tenant_controller.go

```diff
@@ -239,22 +248,22 @@ func (tc *TenantController) syncTenant(tenantName string) (err error) {
 	}
 
 	// handling the addition of a tenant
-	if err, done := tc.createNamespaces(tenantName); !done {
+	if err, done := tc.syncNamespaces(tenantName); !done {
```

Comment:
i guess this is originally based on the namespace controller code, hence the naming of the function as createNamespaces(). they should be changed to syncTenants() in the tenant controller.